src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,38 @@
+@import "var.scss";
+
+.watch {
```

Comment:
We have a requirement - The starting position of the hands should be at 0:00.
Now it starts at 6 o'clock
Please, update your styles